back/tests/cmd/openbsrv-integration/integration_test.go

```diff
@@ -141,11 +213,17 @@ func userSvcAddAndFndUsersFn(ctx context.Context, conn *grpc.ClientConn, clnt pb
 			got.XXX_unrecognized = []byte{}
 			got.XXX_sizecache = 0
 
+			for i := 0; i < len(got.Roles); i++ {
+				got.Roles[i].Id = ""
+				got.Roles[i].XXX_unrecognized = []byte{}
+				got.Roles[i].XXX_sizecache = 0
```

Comment:
Is this done for many types? If so, we might want to make a magical helper that uses reflection to do this.